integration-test/common/src/test/java/org/apache/uniffle/test/CoordinatorGrpcTest.java

```diff
@@ -121,11 +122,29 @@ public void getShuffleRegisterInfoTest() {
 
   @Test
   public void getShuffleAssignmentsTest() throws Exception {
-    String appId = "getShuffleAssignmentsTest";
+    final String appId = "getShuffleAssignmentsTest";
     CoordinatorTestUtils.waitForRegister(coordinatorClient,2);
+    // When the shuffleServerHeartbeat Test is completed before the current test,
+    // the server's tags will be [ss_v4, GRPC_NETTY] and [ss_v4, GRPC], respectively.
+    // We need to remove the first machine's tag from GRPC_NETTY to GRPC
+    shuffleServers.get(0).stopServer();
+    ShuffleServerConf shuffleServerConf = shuffleServers.get(0).getShuffleServerConf();
+    shuffleServerConf.setInteger("rss.rpc.server.port", SHUFFLE_SERVER_PORT + 3);
+    shuffleServerConf.setInteger("rss.jetty.http.port", 18089);
+    shuffleServerConf.setInteger(ShuffleServerConf.NETTY_SERVER_PORT, -3);
```

Comment:
I need to reset netty's port number to a negative number.

Reply:
I think you can get the `RssConf#settings` by refection and remove the config options which you don't need.